### ISSUE TITLE
Screen reader is announcing the incorrect number of Tabs number as 7, even though there are only 5 Tabs available: A11y_ML.NET Model Builder_Column Settings_Screenreader.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3029]
### GitHubTags
#A11ySev2; #A11yTCS;#DesktopApp; #Win32;#Model Builder - Machine Learning for .NET Developers;#BM-MlnetModelBuilder-Win32-March25; #Info and Relationships; #Win32;#A11yMAS;#WCAG1.3.1;#Element:List;#NVDA;#JAWS;#ScreenReader;

### Actual Result:
The screen reader is announcing the incorrect number of Tabs as 7, even though there are only 5 Tabs are avialable.

### Expected Result:
The screen reader should correctly announce the Tabs number as 5, for example: 2 of 5, 3 of 5.

### Environment Details:
Application Name: VS Setup
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.3476
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.13.3

### Repro Steps:
**Prerequisites:**
Install latest Visual Studio 2022 with at least one of the .NET workloads (ASP.NET and web development, .NET desktop development, or .NET Core cross-platform development) and/or the ML.NET Model Builder component selected.

1. Launch visual studio.
2. Turn on NVDA/JAWS.
3. Create a new .NET project or open an existing .NET project.
4. Right-click on the project in the solution explorer and Add > Machine Learning.
5. Tab till scenario and select a scenario.
6. Select training environment (Select local(CPU).
7. Tab till next step and press enter, "select file from the data source type and add a file".
8. Tab till advanced data options and press enter.
9. Tab till advanced training options and press enter.
10. Verify that the screen reader should announce the correct Tabs number as 5, ex- 2 of 5 3 of 5.

### User Impact:
Visually impaired users who rely on screen readers may experience confusion because the screen reader is incorrectly announcing the number of tabs as 7. This discrepancy can lead to misunderstandings and navigation difficulties for these users.

### Attachment:

https://github.com/user-attachments/assets/46b165ed-33da-4b48-bb97-687d2a52f358

**Comment by AnKushSingh05** [comment]
Issue is not fixed in below version.
Microsoft Visual Studio Enterprise 18 (64-bit) - main
Version 18.0.0 Preview 1.0 [10613.167.main]

![Image](https://github.com/user-attachments/assets/5abbda18-a1e7-427c-8ed5-0e897f3b30e3)

**Comment by AnKushSingh05** [comment]
Issue is still not fixed in version 17.19.2.2536801, NVDA continues to announce 7 tabs, although only 5 tabs are currently available.

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/d5983e09-ebe5-4382-b8e1-f199886cdf1d" />
